### ISSUE TITLE
Queue beginTransaction() issue

### ISSUE THREAD

**Issue by sskl** [laravel/framework#15927]
- Laravel Version: 5.3.18
- PHP Version: 7.0.3
- Database Driver & Version: MySQL Ver 15.1 Distrib 10.1.10-MariaDB, for Win32 (AMD64)

### Description:
When i run `php artisan queue:listen`, gave below exception.

> Symfony\Component\Debug\Exception\FatalThrowableError: Call to undefined method Closure::beginTransaction() in laravel/framework/src/Illuminate/Database/Connection.php:606

When changed `$this->pdo->beginTransaction();` to `$this->getPdo()->beginTransaction();` on Connection.php line 606 queue runs successfully.

**Comment by themsaid** [comment]
Full stack trace please.

**Comment by sskl** [comment]
```
Symfony\Component\Debug\Exception\FatalThrowableError: Call to undefined method Closure::beginTransaction() in /var/www/vhosts/site/vendor/laravel/framework/src/Illuminate/Database/Connection.php:606

Stack trace:
#0 /var/www/vhosts/site/vendor/laravel/framework/src/Illuminate/Queue/DatabaseQueue.php(175): Illuminate\Database\Connection->beginTransaction()
#1 /var/www/vhosts/site/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(173): Illuminate\Queue\DatabaseQueue->pop('default')
#2 /var/www/vhosts/site/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(143): Illuminate\Queue\Worker->getNextJob(Object(Illuminate\Queue\DatabaseQueue), 'default')
#3 /var/www/vhosts/site/vendor/laravel/framework/src/Illuminate/Queue/Worker.php(76): Illuminate\Queue\Worker->runNextJob('database', 'default', Object(Illuminate\Queue\WorkerOptions))
#4 /var/www/vhosts/site/vendor/laravel/framework/src/Illuminate/Queue/Console/WorkCommand.php(100): Illuminate\Queue\Worker->daemon('database', 'default', Object(Illuminate\Queue\WorkerOptions))
#5 /var/www/vhosts/site/vendor/laravel/framework/src/Illuminate/Queue/Console/WorkCommand.php(83): Illuminate\Queue\Console\WorkCommand->runWorker('database', 'default')
#6 [internal function]: Illuminate\Queue\Console\WorkCommand->fire()
#7 /var/www/vhosts/site/bootstrap/cache/compiled.php(1272): call_user_func_array(Array, Array)
#8 /var/www/vhosts/site/vendor/laravel/framework/src/Illuminate/Console/Command.php(169): Illuminate\Container\Container->call(Array)
#9 /var/www/vhosts/site/vendor/symfony/console/Command/Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 /var/www/vhosts/site/vendor/laravel/framework/src/Illuminate/Console/Command.php(155): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 /var/www/vhosts/site/vendor/symfony/console/Application.php(820): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 /var/www/vhosts/site/vendor/symfony/console/Application.php(187): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Queue\Console\WorkCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#13 /var/www/vhosts/site/vendor/symfony/console/Application.php(118): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#14 /var/www/vhosts/site/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(122): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#15 /var/www/vhosts/site/artisan(33): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 {main}
```

**Comment by themsaid** [comment]
This change isn't released yet, are you sure you're running `5.3.18` and not pulling directly from the `5.3` branch?

**Comment by GrahamCampbell** [comment]
Please run `composer show -i`. This might be confusing since the version constant in the application class is never updated to be -dev between tags.

**Comment by themsaid** [comment]
It's now solved on the 5.3 branch
